std_msgs/Header header

uint32 t_us
uint16 seq
uint8 flags

float32 flex_adc_f
float32 flex_norm
float32 alpha_used